examples/pytorch/eeg-gcnn/main.py

```diff
@@ -13,6 +15,14 @@
 from sklearn.model_selection import train_test_split
 from torch.utils.data import WeightedRandomSampler
 
+
+def load_memory_mapped_array(file_name):
+    # Since `file_name` is not always byte-aligned,
+    # let's fix this to avoid the warning.
```

Comment:
```suggestion
    # Due to a legacy problem related to memory alignment in joblib [1], the
    # data provided in the example may not be byte-aligned. This can be risky
    # when loading with mmap_mode. To fix the issue, load and re-dump the data.
    # [1] https://joblib.readthedocs.io/en/latest/developing.html#release-1-2-0
```

Reply:
DONE. Thanks for the very good suggestions!